cluster/stone-prd-rh01/tenants/rhtap-has-tenant/integration_test_scenario.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: appstudio.redhat.com/v1beta1
+kind: IntegrationTestScenario
+metadata:
+  name: application-service-enterprise-contract
```

Comment:
wrong name

Reply:
The guide said `<application>-enterprise-contract` for the name here, so I went with `application-service-enterprise-contract` since that's the name of the application: 
<img width="1474" alt="Screenshot 2024-01-18 at 3 03 42 PM" src="https://github.com/redhat-appstudio/tenants-config/assets/6880023/01cef216-55e2-4632-8557-25099628116c">

Should it be another value @scoheb?

Reply:
You are right. My bad. Sorry!